steps/unity-install/0.1.0/step.yml

```diff
@@ -0,0 +1,92 @@
+title: Unity Install
+summary: |
+  Downloads and installs Unity packages
+description: |
+  Downloads and installs Unity packages
+website: https://github.com/bdrelling/bitrise-step-unity-install
+source_code_url: https://github.com/bdrelling/bitrise-step-unity-install
+support_url: https://github.com/bdrelling/bitrise-step-unity-install/issues
+published_at: 2019-03-21T11:28:54.545386-05:00
+source:
+  git: https://github.com/bdrelling/bitrise-step-unity-install.git
+  commit: a168322102969eb88d649d0d3304e756da26df9c
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- xamarin
```

Comment:
Is there a reason for it supports Xamarin only?

What about:
- ios
- android
- react-native

Reply:
My understanding is that the Xamarin stack will have the necessary components to reduce overhead from installation. Is this not the case? Honestly, there should be a Unity project type in the future to replace this with. I would not think `react-native` should work at all, and `ios` and `android` won't include the `dotnet` SDKs.

Reply:
That's right.